README.md

```diff
@@ -81,9 +81,13 @@ pip install -e homu
 
 1. Copy `cfg.sample.toml` to `cfg.toml`, and edit it accordingly.
 
-2. Create a GitHub account that will be used by Homu. You can also use the
-   existing account. In the account settings, register a new application and
-   generate a new access token.
+2. Create a GitHub account that will be used by Homu. You can also use
```

Comment:
It seems that you're using `tw=70` for Markdown. I don't have any preference on a particular text width, but I'm currently using `tw=80`. Is there a reason why you're using 70? If that is reasonable, we should consider making a transition to 70. If not, it would be better to keep using 80.